api/src/domain/types.ts

```diff
@@ -261,6 +263,16 @@ export interface TaxClearanceCertificateClient {
   unlinkTaxId: (userData: UserData, databaseClient: DatabaseClient) => Promise<UnlinkTaxIdResponse>;
 }
 
+export interface PowerAutomateClient {
+  startWorkflow: (props: { body?: object; headers?: object }) => Promise<AxiosResponse>;
+  health: () => Promise<HealthCheckMetadata>;
+}
+
+export interface EmailClient {
+  sendEmail: (postBody: EmailConfirmationSubmission) => Promise<EmailConfirmationResponse>;
+  health: () => Promise<HealthCheckMetadata>;
+}
+
```

Comment:
[dust] I've gone back and forth on whether or not to leave a comment about this interface. Ultimately, I think I'm landing on the side of "this is fine and we can adapt as necessary in the future if things change", but creating an interface called `EmailClient` is a bold move, cotton.

Ultimately, I think it's the right interface for us at this time, and you've done good work thinking through turning it into a factory. I just need to check myself on why I initially bristled at seeing something called `EmailClient`.